cs-oracle-cloud/src/main/java/io/cloudslang/content/oracle/oci/actions/instances/ListInstances.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * (c) Copyright 2020 Micro Focus, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudslang.content.oracle.oci.actions.instances;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType;
+import com.jayway.jsonpath.JsonPath;
+import io.cloudslang.content.constants.OutputNames;
+import io.cloudslang.content.constants.ResponseNames;
+import io.cloudslang.content.constants.ReturnCodes;
+import io.cloudslang.content.oracle.oci.entities.inputs.OCICommonInputs;
+import io.cloudslang.content.oracle.oci.entities.inputs.OCIInstanceInputs;
+import io.cloudslang.content.oracle.oci.services.models.instances.InstanceImpl;
+import io.cloudslang.content.oracle.oci.utils.*;
+import io.cloudslang.content.utils.StringUtilities;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+
+import static io.cloudslang.content.constants.OutputNames.EXCEPTION;
+import static io.cloudslang.content.constants.OutputNames.RETURN_RESULT;
+import static io.cloudslang.content.httpclient.entities.HttpClientInputs.*;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+import static org.apache.commons.lang3.StringUtils.join;
+
+public class ListInstances {
+
+    @Action(name =Constants.ListInstancesConstants.LIST_INSTANCES_OPERATION_NAME ,
```

Comment:
it should be like this 
```suggestion
    @Action(name =LIST_INSTANCES_OPERATION_NAME ,
```

Reply:
Check all classes.